sql/core/src/test/scala/org/apache/spark/sql/streaming/test/DataStreamReaderWriterSuite.scala

```diff
@@ -815,3 +815,26 @@ class DataStreamReaderWriterSuite extends StreamTest with BeforeAndAfter {
     }
   }
 }
+
+class DataStreamTableAPISuite extends StreamTest with BeforeAndAfter {
```

Comment:
nit: could you move this to a new file when you adds more tests?

Reply:
Sure, adding more tests now.